Patches/Caffe/Patch.cmake

```diff
@@ -1,6 +1,20 @@
-
-file(COPY
-  ${Caffe_patch}/Dependencies.cmake
-  DESTINATION
-  ${Caffe_source}/cmake/
-  )
+if(WIN32)
+	file(COPY
+	  ${Caffe_patch}/WindowsDownloadPrebuiltDependencies.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/)
+	file(COPY
+	  ${Caffe_patch}/FindGFlags.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/modules/)
+	file(COPY
+	  ${Caffe_patch}/CMakeLists.txt
+	  DESTINATION
+	  ${Caffe_source}/)
+else()
+	file(COPY
+	  ${Caffe_patch}/Dependencies.cmake
+	  DESTINATION
+	  ${Caffe_source}/cmake/
+	  )
```

Comment:
Well, this file does, at least. :) As I mentioned in the other comments, I used notepad++ for this and the default settings bork spacing. Right now I'm running through the build on a couple of platforms to see where it runs and where/why it doesn't, but running through and fixing up the spacing problems is first on my list after that's done.